ammo-be/src/routers/bulletRouter/__tests__/ws.test.ts

```diff
@@ -78,4 +79,61 @@ describe('[WS] BulletRouter', () => {
             done();
         });
     });
+
+    it('The recording state must be initialized to true', () => {
+        expect(isRecording).toBe(true);
+    });
+
+    it('Should change the recording state to false', (done) => {
```

Comment:
Tests descriptions shoud match : 'what we do - what we expect'

Here : 'Toggling the recorder off - recording state should be false'